src/libs/actions/Report.js

```diff
@@ -663,14 +664,15 @@ function addAction(reportID, text, file) {
  *
  * @param {Number} reportID
  * @param {Number} sequenceNumber
+ * @param {Boolean} ignoreOrder
  */
-function updateLastReadActionID(reportID, sequenceNumber) {
+function updateLastReadActionID(reportID, sequenceNumber, ignoreOrder = false) {
     const currentMaxSequenceNumber = reportMaxSequenceNumbers[reportID];
-    if (sequenceNumber < currentMaxSequenceNumber) {
+    if (!ignoreOrder && sequenceNumber < currentMaxSequenceNumber) {
         return;
     }
 
-    setLocalLastRead(reportID, sequenceNumber);
+    setLocalLastRead(reportID, sequenceNumber, currentMaxSequenceNumber);
```

Comment:
I don't think we need to add `currentMaxSequenceNumber` here. We're passing this to a method in the same scope that can access the same stuff.